src/main/java/org/elasticsearch/plugin/readonlyrest/settings/ConfigMalformedException.java

```diff
@@ -15,10 +15,13 @@
  *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
  */
 
-package org.elasticsearch.plugin.readonlyrest.acl.blocks.rules;
+package org.elasticsearch.plugin.readonlyrest.settings;
 
 public class ConfigMalformedException extends RuntimeException {
   public ConfigMalformedException(String msg) {
     super(msg);
   }
+  public ConfigMalformedException(String msg, Throwable t) {
```

Comment:
We should unify the concept of "Config" with the "Setting", let's choose one and stick with it